### PR TITLE
Eng 12351 - update voltadmin shutdown script logic to take into kit type

### PR DESCRIPTION
**Pull request by hhakimi53**
Update "voltadmin shutdown" logic to take into account kit type (community or not) before invoking features which are community specific.  For example features like export queise, snapshot save and command logs will not be invoked if it's community version.

Output of "voltdbadmin shutdown": 

Community Version:

```
$ voltadmin shutdown 
INFO: Cluster shutdown in progress.
INFO: Preparing for shutdown...
INFO: The cluster is paused prior to shutdown.
INFO: Completing outstanding client transactions...
INFO: Completing outstanding importer requests...
INFO: Shutting down the cluster...
Status: -1
Information: Connection broken

$ voltadmin shutdown -s
WARNING: Snapshots not supported in the community edition. The --save option is being ignored.
INFO: Cluster shutdown in progress.
INFO: Preparing for shutdown...
INFO: The cluster is paused prior to shutdown.
INFO: Completing outstanding client transactions...
INFO: Completing outstanding importer requests...
INFO: Shutting down the cluster...
Status: -1
Information: Connection broken

$
```

Enterprise Version:

```
$ voltadmin shutdown
INFO: Cluster shutdown in progress.
INFO: Preparing for shutdown...
INFO: The cluster is paused prior to shutdown.
INFO: Writing out all queued export data...
INFO: Completing outstanding client transactions...
INFO: Completing outstanding importer requests...
INFO: Completing outstanding Command Log transactions...
INFO: All transactions have been made durable.
INFO: Shutting down the cluster...
Status: -1
Information: Connection broken

$ voltadmin shutdown -s
INFO: Cluster shutdown in progress.
INFO: Preparing for shutdown...
INFO: The cluster is paused prior to shutdown.
INFO: Writing out all queued export data...
INFO: Completing outstanding client transactions...
INFO: Completing outstanding importer requests...
INFO: Completing outstanding Command Log transactions...
INFO: All transactions have been made durable.
INFO: Completing outstanding DR consumer transactions...
INFO: Starting resolution of external commitments...
INFO: Completing outstanding exporter transactions...
INFO: All exporter transactions have been processed.
INFO: Completing outstanding DR producer transactions...
INFO: All DR producer transactions have been processed.
INFO: Saving a final snapshot, The cluster will shutdown after the snapshot is finished...
Status: -1
Information: Connection broken

$ 
```

### CODE REVIEW

File: lib/python/voltcli/voltadmin.d/shutdown.py [hunk 1]
```diff
@@ -36,6 +37,13 @@ def shutdown(runner):
     shutdown_params = []
     columns = []
     zk_pause_txnid = 0
+
+    communityVersion = isCommunityVersion(runner)
+    utility.debug("Community version? ", communityVersion)
```
Comment (akhanzode) [inline lib/python/voltcli/voltadmin.d/shutdown.py]: Whats the purpose of this debug?

File: lib/python/voltcli/voltadmin.d/shutdown.py [hunk 2]
```diff
@@ -78,3 +92,11 @@ def shutdown(runner):
             runner.abort(actionMessage)
     response = runner.call_proc('@Shutdown', columns, shutdown_params, check_status = False)
     print response
+
+
+def isCommunityVersion(runner):
+    response = runner.call_proc('@SystemInformation', [VOLT.FastSerializer.VOLTTYPE_STRING], ['OVERVIEW'])
+    for tuple in response.table(0).tuples():
```
Comment (akhanzode) [inline lib/python/voltcli/voltadmin.d/shutdown.py]: What if server is not up and you dont get response? How do you handle this?

File: lib/python/voltcli/voltadmin.d/shutdown.py (hunk 1 above)
Comment (hhakimi53) [inline lib/python/voltcli/voltadmin.d/shutdown.py]: Have added for debugging purpose so as to see what value it's picking up. I can clean it up.

File: lib/python/voltcli/voltadmin.d/shutdown.py (hunk 2 above)
Comment (hhakimi53) [inline lib/python/voltcli/voltadmin.d/shutdown.py]: If the server is not up, the call_proc() call will fail and cli will exit. The error message that it gives for this and other proc calls when it's invoked when server is down is: "ERROR: error Exception: [Errno 111] Connection refused".

**Pull request merged**